misk-testing/src/test/kotlin/misk/testing/NestedTestsTest.kt

```diff
@@ -0,0 +1,52 @@
+package misk.testing
+
+import misk.inject.KAbstractModule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@MiskTest(startService = false)
```

Comment:
Might be good to have a test or update this one to `true` to ensure this is still working

Reply:
updated to include a service ‼️